website/content/en/docs/advanced-topics/scorecard/scorecard.md

```diff
@@ -30,26 +30,28 @@ require if the tests are designed for resource creation.
 
 ## Running the Scorecard
 
-1. Define a scorecard configuration file `config.yaml`.  A sample
-configuration file [config.yaml][sample-config] is found within
-the SDK repository. See the [config file section](#config-file) for an explaination
-of the configuration file format.  Unless you are executing custom
-tests, you can just copy the provided example configuration file
-into your project.
-2. Place the scorecard configuration file within your project
-bundle directory at the following location `tests/scorecard/config.yaml`.
-You can override the default location of the configuration file
-by specifying the `--config` flag.
-3. Execute the [`scorecard` command][cli-scorecard]. See the
-[command args section](#command-args) for an overview of command invocation.
+1. Generate [bundle][quickstart-bundle] manifests and metadata for your Operator.
```

Comment:
These steps are a bit wonky right now because `generate bundle` will only write `COPY local/scorecard tests/scorecard/` if `local/scorecard` exists, but if that path exists then `bundle/metadata/annotations.yaml` will contain
```yaml
annotations:
  ...
  operators.operatorframework.io.test.config.v1: tests/scorecard/
```
which is incorrect. This creates a chicken and egg problem that can only be resolved if a user first writes `tests/scorecard/config.yaml` then copies into `bundle/tests/scorecard/config.yaml` after running `make bundle`.